README.md

```diff
@@ -1,16 +1,22 @@
-# Overview
+# Carbon.Cryptography
+
+## Overview
 
 The "Carbon.Cryptography" module has functions that help manage X509 certificates and working with secure strings.
 
 It started out as part of the [Carbon](http://get-carbon.org) module. But when that module become too big and unwieldy,
 its security-related functionality was moved into this module.
 
-# System Requirements
+## System Requirements
 
 * Windows PowerShell 5.1 and .NET 4.6.1+
 * PowerShell Core 6+
 
-# Installing
+## Dependencies
+
+Carbon.Cryptography has a private, nested dependency on PureInvoke 1.0.1.

```

Comment:
I don't see a PureInvoke dependency?

Reply:
It's in both Carbon.Accounts and Carbon.Security. I want to call it out in case users get type mismatch problems with having multiple different versions loaded.

Reply:
That makes sense 👍